### PR TITLE
Adding Replicated information to Terraform Enterprise security model

### PR DESCRIPTION
**Pull request by eastebry**
Including blurb about Replicated, TFE's third-party software packaging solution

### CODE REVIEW

File: content/source/docs/enterprise/system-overview/security-model.html.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ Infrastructure admins are responsible for all aspects of reliability and availab
 
 Unlike Terraform Cloud, Terraform Enterprise performs all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services. However, Terraform Enterprise does not perform any egress filtering, so Terraform runs can still access available network resources.
 
+### Terraform Enterprise Relies on Third Party Software for Licensing, Delivery, Installation, and Management
+Terraform Enterprise is built on top of a software platform developed by [Replicated](https://www.replicated.com/). The components neccesary for installing Terraform Enterprise are hosted by Replicated, and software developed by Replicated is used for bootstrapping, configuring, and managing every Terraform Enterprise deployment.
```
Comment (chair6) [inline content/source/docs/enterprise/system-overview/security-model.html.md:52]:
```suggestion
Terraform Enterprise is built on top of a software platform developed by [Replicated](https://www.replicated.com/). The components neccesary for installing Terraform Enterprise are hosted by Replicated, and software developed by Replicated is used for bootstrapping, configuring, and managing every Terraform Enterprise deployment. For more information, see [Security at Replicated](https://www.replicated.com/security/).
```

**Review by chair6 — COMMENTED** [review]
LGTM!  Good to get this explicitly covered in the threat model.  I added a link to Replicated's /security/, but happy to skip this change if it doesn't make sense...

**Review by aaron-lane — APPROVED** [review]
LGTM 🔐

File: content/source/docs/enterprise/system-overview/security-model.html.md [hunk 2]
```diff
@@ -48,6 +48,9 @@ Infrastructure admins are responsible for all aspects of reliability and availab
 
 Unlike Terraform Cloud, Terraform Enterprise performs all Terraform operations in Docker containers on the Terraform Enterprise host. The containers are assigned to an isolated Docker network to prevent them from communicating with Terraform Enterprise backend services. However, Terraform Enterprise does not perform any egress filtering, so Terraform runs can still access available network resources.
 
+### Terraform Enterprise Relies on Third Party Software for Licensing, Delivery, Installation, and Management
+Terraform Enterprise is built on top of a software platform developed by [Replicated](https://www.replicated.com/). The components neccesary for installing Terraform Enterprise are hosted by Replicated, and software developed by Replicated is used for bootstrapping, configuring, and managing every Terraform Enterprise deployment. For more information, see [Security at Replicated](https://www.replicated.com/security/).
```
Comment (chair6) [inline content/source/docs/enterprise/system-overview/security-model.html.md:52]:
```suggestion
Terraform Enterprise is built on top of a software platform developed by [Replicated](https://www.replicated.com/). The components necessary for installing Terraform Enterprise are hosted by Replicated, and software developed by Replicated is used for bootstrapping, configuring, and managing every Terraform Enterprise deployment. For more information, see [Security at Replicated](https://www.replicated.com/security/).
```

**Review by chair6 — APPROVED** [review]
🚀

**Pull request merged**